src/app/editor-toolbar/editor-toolbar-save/editor-toolbar-save.component.ts

```diff
@@ -0,0 +1,24 @@
+import {
+    Component,
+    Input
+} from '@angular/core';
+import { RecordService } from '../../shared/services';
+
+@Component({
+  selector: 're-editor-toolbar-save',
+  templateUrl: './editor-toolbar-save.component.html'
+})
+
+export class EditorToolbarSaveComponent {
+  @Input() record: Object;
+  private errorMap: Object;
+
+  constructor(private recordService: RecordService) {
+
+  }
+
+  onClickSave(event: Object) {
+    this.recordService.saveRecord(this.record)
+    .subscribe(resp => this.errorMap = resp['errorMap']);
```

Comment:
Does the response include `errorMap` ?

And if it did, is it enough assigning it to the private `errorMap` for it show up in the UI?